package.json

```diff
@@ -31,23 +32,20 @@
     "browserify": "^16.5.0",
     "jest": "^24.9.0",
     "prettier": "^1.18.2",
-    "safe-buffer": "^5.2.0",
     "ts-jest": "^24.1.0",
     "tslint": "^5.20.0",
     "tslint-config-prettier": "^1.18.0",
     "typescript": "^3.8.2"
   },
   "dependencies": {
     "@types/cashaddrjs": "^0.3.0",
+    "base32.js": "^0.1.0",
     "bech32": "^1.1.3",
-    "bs58check": "^2.1.2",
-	"blakejs": "^1.1.0",
+    "blakejs": "^1.1.0",
     "cashaddrjs": "^0.3.8",
-    "eosjs-ecc": "^4.0.7",
-    "nem-sdk": "^1.6.7",
-    "ripple-address-codec": "^4.0.0",
-    "rskjs-util": "^1.0.3",
-    "stellar-base": "^2.1.2",
+    "crc": "^3.8.0",
+    "crypto-addr-codec": "^0.1.1",
```

Comment:
Can you please pin this to a specific version?

Reply:
sure, can i include a yarn.lock file? or just by using the version would be okay?

Reply:
Just using the version is fine.